api/rest/restcore/issues_rest.php

```diff
@@ -297,3 +302,40 @@ function rest_issue_monitor_add( \Slim\Http\Request $p_request, \Slim\Http\Respo
 	return $p_response->withStatus( HTTP_STATUS_CREATED, "Users are now monitoring issue $t_issue_id" )->
 		withJson( array( 'issues' => array( $t_issue ) ) );
 }
+
+/**
+ * Add issue relationship.
+ *
+ * @param \Slim\Http\Request $p_request   The request.
+ * @param \Slim\Http\Response $p_response The response.
+ * @param array $p_args Arguments
+ * @return \Slim\Http\Response The augmented response.
+ */
+function rest_issue_relationship_add( \Slim\Http\Request $p_request, \Slim\Http\Response $p_response, array $p_args ) {
+	$t_relationship_info = $p_request->getParsedBody();
```

Comment:
I would suggest support the following payload format.  Note the following:

- Source issue id is extracted from URL.
- Target issue id from payload.
- Relationship type is referenced by logical name (use same names used in output of get issue).

```
URL: https://.../api/rest/issues/1/relationships/
Method: POST
{
  relationship_type: "related-to",
  issue_id: 123
}
```